CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ if(LLVM_SPIRV_BUILD_EXTERNAL)
       ${LLVM_SPIRV_VERSION}
     LANGUAGES
       CXX
+      C
```

Comment:
Why do we need C language here?

Reply:
As I understand, it is just the requirement for HandleLLVMOptions

https://github.com/llvm/llvm-project/blob/main/llvm/cmake/modules/HandleLLVMOptions.cmake#L11